TimberWinR/Outputs/Redis.cs

```diff
@@ -100,11 +109,19 @@ public override JObject ToJson()
         public RedisOutput(TimberWinR.Manager manager, Parser.RedisOutput ro, CancellationToken cancelToken)
             : base(cancelToken, "Redis")
         {
+            // Last QUEUE_SAMPLE_SIZE queue samples (spans timestamp * 10)
+            _sampleQueueDepths = new long[QUEUE_SAMPLE_SIZE];
+            _sampleCountIndex = 0;
             _redisDepth = 0;
             _batchCount = ro.BatchCount;
+            _maxBatchCount = ro.MaxBatchCount;
+            // Make sure maxBatchCount is larger than batchCount
+            if (_maxBatchCount < _batchCount)
```

Comment:
This could have issues if the batch count is more than 10 times the size of the max batch count.